corehq/apps/ota/tests/digest_restore.py

```diff
@@ -5,6 +5,13 @@
 from django_digest.test import Client
 
 
+def setupModule():
+    from unittest import SkipTest
+    raise SkipTest("this is broken, hasn't been run by django test runner. Error: "
```

Comment:
Oh, right. Sorry I thought this was the one in `performance_tests.py`. Yes,
please remove that import or convert the module level skip to test level
skips using `@skip`

On 27 May 2016 at 11:21, Sravan Reddy <notifications@github.com> wrote:

> In corehq/apps/ota/tests/digest_restore.py
> <https://github.com/dimagi/commcare-hq/pull/11705#discussion_r64879100>:
>
> > @@ -5,6 +5,13 @@
> >  from django_digest.test import Client
> >
> >
> > +def setupModule():
> > +    from unittest import SkipTest
> > +    raise SkipTest("this is broken, hasn't been run by django test runner. Error: "
>
> Looks like we do
> https://github.com/dimagi/commcare-hq/blob/master/corehq/apps/ota/tests/__init__.py#L1
> Should I remove it?
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/dimagi/commcare-hq/pull/11705/files/b2da58600d8045173595b670337b715b48466369#r64879100>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/AAPPBpXLlgYP6Tv479gerCMRJLuMsNNaks5qFrehgaJpZM4Ikbr9>
> .
>



-- 
Simon Kelly
Senior Engineer | Dimagi South Africa